<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplashScreen</class>
 <widget class="QMainWindow" name="SplashScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="main_frame">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>660</width>
      <height>380</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	border: 2px solid rgb(156, 156, 156);
	background-color: rgb(255, 255, 255);
	border-radius: 10px;
 	
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>290</y>
       <width>581</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar {
	background-color: rgb(98, 114, 164);
	color: rgb(200, 200, 200);
	border-style: none;
	border-radius: 10px;
	text-align: center;
}

QProgressBar::chunk {
	border-radius: 10px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0.494, x2:1, y2:0.517, stop:0 rgba(254, 121, 199, 255), stop:1 rgba(170, 85, 255, 255));
}</string>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="loading_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>641</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	color: rgb(98, 114, 164);
	border: none
}</string>
     </property>
     <property name="text">
      <string>loading...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QFrame" name="logo_frame">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>371</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	
	image: url(eduneering_logo2.png);
	border: none
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QLabel" name="version_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>641</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	color: rgb(125, 125, 125);
	border: none;
}
</string>
     </property>
     <property name="text">
      <string>Version 0.1.2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="info_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>641</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border: none;
}</string>
     </property>
     <property name="text">
      <string>Görüntü İşlemeli Çekme Deney Düzeneği</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
